### DISCUSSION TITLE
Edited tracks verses Global track edits

### DISCUSSION THREAD

**Discussion by gworley** [web-scrobbler/web-scrobbler#3081] [Q&A]
Edited tracks verses Global track edits.  The options say that:
>This extension stores edited tracks.

However I have done many edits (in the past 3 days I have done over 300 edits) in last.fm.  The extension is reporting I have done none.  How do I get the tracks to be added to the Edit function?

Also I do I correct bad global edit tracks .  I was playing a The Sister of Mercy track and it had in a - where the Remaster 2018 after the - was removed however the space- wasn't.

Thanks in advance,
George

**Comment by Infinitay** [comment]
A little confused on this discussion. I believe edits you make through the extension are local and not saved to Web Scrobbler's servers nor pushed to LastFM. That being said, any edits you submit on LastFM will also not be reflected in this extension.

~~I actually came here on GitHub to see if anyone suggested tracking edits so that they are done globally but couldn't find anything.~~ #2157

**Comment by gworley** [comment]
Sorry I was confused too.  I figured that out.

The thing I wish that the extension did is cache scrobbles as with last.fm
I am constantly running in the 3,000 day last.fm limit.

Thanks,
George

**Reply by Infinitay** [reply]
> I am constantly running in the 3,000 day last.fm limit.

Assuming at worst case a song is 1 minute long, that's over two days of playtime in a single day...

What the heck are you doing lol

**Comment by gworley** [comment]
I have 3 other people in my family that use the same music library on my 12
TB (mirrored) NAS drive that I do so instead of creating 4 accounts with
last.fm (especially since I like the features of last.fm Pro).  My family
doesn't really like TV and is listening to music all the time.  My son
cannot sleep with music going and when he is out and about he uses a 500gb
iPod Classic (that was originally 130gb that I modified to get more music
on it).  My daughter doesn't listen to that much music, however she has an
8gb iPod Touch that I wish I could modify. My other half listens to Spotify
all the time which if you are listening to the app with the last.fm
application enabled will scrobble if the limit has been obtained. I usually
listen to music on my Mac or my 256gb 6th generation iPod Touch when I am
outside my home.  I know some of this is old school but it saves money
listening to music for us.  Mostly we like the same type of music.  My
iTunes/Apple Music account is a family plan.

George

**Reply by Infinitay** [reply]
Oh wow okay. Well I would suggest looking into the manual scrobbler [Last.fm-Scrubbler-WPF](https://github.com/SHOEGAZEssb/Last.fm-Scrubbler-WPF) if you were to have your play history. ~~If my memory serves me correctly, there is a feature on that would scrobble as many tracks as it could for the day, and then it would save the rest for when the scrobble limit is reset the following day.~~ My mistake, it seems like it was a feature they were looking to implement, though I'm not sure that they have entirely. According to the GitHub README it doesn't seem like it's been fully implemented yet. Sorry.

Hope you're able to use it or that it helps in any way.
